app/src/main/java/org/p2p/wallet/App.kt

```diff
@@ -78,6 +96,10 @@ class App : Application() {
         AppCompatDelegate.setDefaultNightMode(AppCompatDelegate.MODE_NIGHT_FOLLOW_SYSTEM)
     }
 
+    private fun setupWorkManager() {
+        WorkManager.initialize(this, Configuration.Builder().build())
```

Comment:
Configuration.Builder().build() -> getWorkManagerConfiguration()

Reply:
good catch, thank you